Bio/motifs/meme.py

```diff
@@ -172,13 +157,23 @@ def __read_datafile(record, handle):
         line = next(handle)
     except StopIteration:
         raise ValueError("Unexpected end of stream: Expected to find line starting with 'DATAFILE'")
-    if not line.startswith('DATAFILE'):
-        raise ValueError("Line does not start with 'DATAFILE':\n%s" % line)
+    if not line.startswith('PRIMARY SEQUENCES'):
+        raise ValueError("Line does not start with 'PRIMARY SEQUENCES':\n%s" % line)
     line = line.strip()
-    line = line.replace('DATAFILE= ', '')
+    line = line.replace('PRIMARY SEQUENCES= ', '')
     record.datafile = line
+    
+    try:
+        line = next(handle)
+    except StopIteration:
+        raise ValueError("Unexpected end of stream: Expected to find line starting with 'DATAFILE'")
+    if not line.startswith('CONTROL SEQUENCES'):
+        raise ValueError("Line does not start with 'CONTROL SEQUENCES':\n%s" % line)
+    line = line.strip()
+    line = line.replace('CONTROL SEQUENCES= ', '')
+    record.control_seq = line
 
-
+    
```

Comment:
Style issue - you need to remove the trailing white space here. This will fail our automated ``flake8`` checks, have a look at this if you're not familiar with this tool:

https://github.com/biopython/biopython/blob/master/CONTRIBUTING.rst